.travis.yml

```diff
@@ -1,30 +1,26 @@
 language: rust
 
-env:
```

Comment:
This env is replaced if one is defined in the matrix. This meant that previously this flag was only set on Windows builds.